i18n/id-ID.js

```diff
@@ -53,33 +53,34 @@ module.exports = {
         'Nuxt.js adalah kerangka kerja minimalis untuk membuat aplikasi Vue.js dengan server side rendering, hot-reloading, static generation, dan banyak lagi!'
     },
     welcome: {
-      title: 'Kerangka Kerja Intuitif {br} {frameworkType}',
+      title: 'Kerangka Kerja {br} {frameworkType} yang Intuitif',
```

Comment:
Slogan Vue.js berbahasa Indonesia, "Kerangka Kerja JavaScript nan Progresif". Mungkin untuk menjaga kekonsistensian, menurut saya slogan untuk Nuxt.js berbasa Indonesia berikut yang cocok:

```suggestion
      title: 'Kerangka Kerja {br} {frameworkType} nan Intuitif',
```